stash/src/model/Transaction.ts

```diff
@@ -0,0 +1,23 @@
+import { Schema } from 'redis-om'
+
+const transactionSchema = new Schema(
+  'transactions',
```

Comment:
Model now has fields only for deposits, fields for withdrawals will be added when withdrawals tracing is implemented